Doc/library/functions.rst

```diff
@@ -1822,6 +1822,30 @@ are always available.  They are listed here in alphabetical order.
               super().method(arg)    # This does the same thing as:
                                      # super(C, self).method(arg)
 
+   .. warning::
```

Comment:
`.. warning` blocks are rendered very dramatically with our current theme, such that I think I'd probably only use them for known security vulnerabilities:

<details>
<summary>Screenshot</summary>

![20E3DAC1-E8FB-4640-A48B-9CF6D6AA2E9F](https://github.com/python/cpython/assets/66076021/e2598ac8-23f0-4fea-88b5-d6267b96a3f1)

</details>

I'd suggest using `.. note` instead.
```suggestion
   .. note::
```

In my experience, this kind of thing comes up relatively rarely, so I'd probably also put the note at the bottom of the docs for super(), rather than in the middle

Reply:
👍